changelog/dZvtWFeQSKO9cn3iVU3GhA.md

```diff
@@ -0,0 +1,4 @@
+audience: users
+level: minor
```

Comment:
Should likely be a major bump since an updated client will not be backwardly compatible with the previous version of the interactive session running on the worker. Chances are _very low_ that this would break for anyone since it's likely not being used, but still good to follow semver